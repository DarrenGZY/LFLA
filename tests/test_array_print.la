function main()
{
    var a[4] = {2, 4, 5, 6};
    vector v1 = [1, 2, 3];
    vector v2 = [2, 3, 4];
    vector v3 = [3, 4, 5];
    vector b[3] = {v1, v2, v3};
    print(b);
}
