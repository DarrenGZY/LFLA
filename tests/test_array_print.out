[array([1, 2, 3]), array([2, 3, 4]), array([3, 4, 5])]
